cmd/notary/keys.go

```diff
@@ -91,6 +94,13 @@ var cmdKeyImportRoot = &cobra.Command{
 	Run:   keysImportRoot,
 }
 
+var cmdKeyRemoveKey = &cobra.Command{
```

Comment:
Can we just do `cmdRemoveKey` or `cmdKeyRemove`? `KeyRemoveKey` just feels so redundant.